src/index.html

```diff
@@ -4,9 +4,324 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#" class="logo">
+        <img
+          src="./images/logo.png"
+          alt="logo"
+          class="logo"
+        >
+      </a>
+      <nav class="navigation">
+        <ul class="navigation__list list">
+          <li class="list__element">
+            <a href="#" class="list__link is-active">Apple</a>
+          </li>
+
+          <li class="list__element">
+            <a href="#" class="list__link">Samsung</a>
+          </li>
+
+          <li class="list__element">
+            <a href="#" class="list__link">Smartphones</a>
+          </li>
+
+          <li class="list__element">
+            <a href="#" 
+              class="list__link"
+              data-qa="nav-hover"
+            >Laptops & Computers</a>
+          </li>
+
+          <li class="list__element">
+            <a href="#" class="list__link">Gadgets</a>
+          </li>
+
+          <li class="list__element">
+            <a href="#" class="list__link">Tablets</a>
+          </li>
+
+          <li class="list__element">
+            <a href="#" class="list__link">Photo</a>
+          </li>
+
+          <li class="list__element">
+            <a href="#" class="list__link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="main">
+      <div class="container-catalog">
+        <section class="card" data-qa="card">
+          <img
+            src="./images/imac.jpeg"
+            alt= "APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+            class="card__image"
+          >
+          <h1 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h1>
+          
+          <h2 class="card__subtitle">Product code: 195434</h2>
+          <div class="card__rating-block">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+    
+              <div class="stars__star"></div>
+    
+              <div class="stars__star"></div>
+    
+              <div class="stars__star"></div>
+    
+              <div class="stars__star"></div>
+            </div>
+            <span class="card__rewiew">Reviews: 5</span>
+          </div>
+          <div class="card__price-block">
+            <span class="card__price-block-name">Price:</span>
+    
+            <span class="card__price-block-value">$2,199</span>
+          </div>
+          <a href="#"
+            class="card__button"
+            data-qa="card-hover">Buy
+          </a>
```

Comment:
indentations

Reply:
done

Reply:
nope
find the checklist from first tasks and check how to fix it